auth/src/main/java/com/firebase/ui/auth/ui/email/EmailLinkFragment.java

```diff
@@ -34,6 +34,9 @@
     private TroubleSigningInListener mListener;
     private ScrollView mTopLevelView;
 
+    // Used to avoid sending a new email when popping off the fragment backstack
+    private boolean mEmailSent;
```

Comment:
Does this survive rotation?

Reply:
Nope, my bad :-1: . I also fixed the case where we go from EmailLinkFragment -> TroubleSigningInFragment -> EmailLinkFragment (by clicking resend), we'd be at EmailLinkFragment and have to press the back twice to get back to the nascar screen.